internal/goeland/fetch/feed.go

```diff
@@ -119,10 +123,9 @@ func fetchFeed(source *goeland.Source, feedLocation string, isFile bool, allowIn
 			hash.Write([]byte(entry.URL))
 			entry.UID = hex.EncodeToString(hash.Sum([]byte{}))
 		}
+		entry.Source = *source

```

Comment:
I think it would be better to store pointer to Source (same for the struct def).

Reply:
Did the change see caa3701